### PR TITLE
Settings redesign

### PR DESCRIPTION
**Pull request by adrena-tyro**

### CODE REVIEW

File: src/components/Settings/Settings.tsx [hunk 1]
```diff
@@ -46,19 +43,39 @@ export default function Settings({
...
-  );
-
-  const priorityFeeAmounts = usePriorityFee();
-
-  const currentPriorityFeeValue = priorityFeeAmounts[priorityFeeOption] || priorityFeeAmounts.medium;
+  const [cookies, setCookies] = useCookies(['solanaExplorer']);
+
+  const [activeSolanaExplorer, setActiveSolanaExplorer] =
+    useState<SolanaExplorerOptions>(
+      cookies.solanaExplorer || 'Solana Explorer',
```
Comment (adrena-orex) [inline src/components/Settings/Settings.tsx:52]: Should check if `cookies.solanaExplorer` is a valid explorer here before loading it.

For example, if we remove one explorer from the list in the future and someone has an old cookie, the app would not function properly.

File: src/components/Settings/Settings.tsx [hunk 2]
```diff
@@ -46,19 +43,39 @@ export default function Settings({
...
+    SOLANA_EXPLORERS_OPTIONS,
+  ) as Array<SolanaExplorerOptions>;
+
+  useEffect(() => {
+    window.adrena.settings.solanaExplorer = activeSolanaExplorer;
+
+    setCookies('solanaExplorer', activeSolanaExplorer);
+  }, [activeSolanaExplorer]);
+
+  const WALLETS = {
```
Comment (adrena-orex) [inline src/components/Settings/Settings.tsx:65]: Should remame WALLETS

File: src/pages/_app.tsx [hunk 3]
```diff
@@ -133,7 +139,12 @@ export default function App(props: AppProps) {
 
     const pythConnection = new Connection(config.pythnetRpc.url, 'processed');
 
-    initializeApp(config, activeRpc.connection, pythConnection).then(() => {
+    initializeApp(
+      cookies.solanaExplorer,
```
Comment (adrena-orex) [inline src/pages/_app.tsx:143]: same, check the cookie here is valid

File: src/types.d.ts [hunk 4]
```diff
@@ -23,13 +23,24 @@ export type GeoBlockingData = {
   allowed: boolean;
 };
 
+export type SolanaExplorerOptions =
+  | 'Solana Explorer'
+  | 'Solscan'
+  | 'Solana Beach';
```
Comment (adrena-orex) [inline src/types.d.ts:29]: Missing SolanaFM?

**Review by adrena-orex — DISMISSED** [review]
Looking neat

**Comment by adrena-orex** [comment]
Buttons feels very long. Maybe make them shorter? Or center the text

Screenshot 2024-10-22 at 07 41 38

File: src/components/Settings/Settings.tsx (hunk 1 above)
Comment (adrena-tyro) [reply src/components/Settings/Settings.tsx:52]: good catch! will fix

**Review by adrena-kino — APPROVED** [review]

File: src/components/Settings/Settings.tsx [hunk 5]
```diff
@@ -46,19 +43,43 @@ export default function Settings({
...
   isGenesis?: boolean;
 }) {
-  const [editCustomRpcUrl, setEditCustomRpcUrl] = useState<string | null>(
-    customRpcUrl,
-  );
-
-  const priorityFeeAmounts = usePriorityFee();
-
-  const currentPriorityFeeValue = priorityFeeAmounts[priorityFeeOption] || priorityFeeAmounts.medium;
+  const [cookies, setCookies] = useCookies(['solanaExplorer']);
```
Comment (Adrena-Corto) [inline src/components/Settings/Settings.tsx:48]: I don't think cookies should be here, we have them all in _app.tsx, probably want them there too, and propagate (See how it's been done for the prio fees for instance)

Comment (Adrena-Corto) [reply src/components/Settings/Settings.tsx:48]: (ps: I did the same mistake, Orex shown me the other day)

**Review by Adrena-Corto — APPROVED** [review]

**Review by Adrena-Corto — APPROVED** [review]

**Comment by adrena-tyro** [comment]
@adrena-orex doesnt look great when i try different solution x( will revisit the design at some point again

**Pull request merged**